src/core/src/sim/mc/moves/mod.rs

```diff
@@ -65,6 +70,76 @@ fn select_molecule(system: &System, moltype: Option<u64>, rng: &mut Box<Rng>) ->
     }
 }
 
+/// This struct keeps track of the number of times a move was called
+/// and how often it was accepted. It also stores the current maximum displacement,
+/// i.e. the magnitude of a particle's translation/rotation or a volume's change in size.
+/// 
+/// We can modify (`update`) the maximum displacement to achieve a target acceptance rate 
+/// of a move based on the running acceptance.
+pub struct Displacement {
+    /// Current maximum displacement
+    delta: f64,
+    /// Current sample range
+    pub delta_range: Range<f64>,
+    /// Number of move calls
+    pub ncalls: u64, 
```

Comment:
`nattempted` (as well as `naccepted`) will be set to zero after an "successful" update. `ncalls` continues to count. Currently, `ncalls` is only used to check if the displacement should be updated (and the information about the total numbers of calls to the move). We could also use it in conjunction with a `nequilibration` variable which automatically switches off updates after a certain number of trials.

We could eliminate `ncalls` and compare the frequency directly to `nattempts` in `update`.